docs/enterprise/console-settings.yaml

```diff
@@ -234,38 +234,38 @@ settings:
 
           1. From the main menu, select **Service Accounts** under **CONFIGURE**. Click the **+ ADD SERVICE ACCOUNT** button:
 
-             ![An empty Service Accounts page](../img/add-service-account.png)
+             ![An empty Service Accounts page](/enterprise/img/add-service-account.png)
 
           1. Service accounts can be unique and exist only for Pomerium, or impersonate directory users from your IdP.
 
              ::::: tabs
              :::: tab Unique
              Give the user a unique ID. Consider referencing the Namespace you're creating it under, for easier reference later. Optionally set an expiration date:
 
-             ![Adding a unique service account](../img/create-service-account.png)
+             ![Adding a unique service account](/enterprise/img/create-service-account.png)
 
              The user ID set here corresponds to the `User` criteria when editing a policy.
              ::::
              :::: tab Impersonated
              You can find your User ID by going to the special endpoint `/.pomerium`, or selecting **Logout** under your user in the upper right hand corner (this will not immediately log you out):
 
-             ![Session Details](../img/user-id.png)
+             ![Session Details](/enterprise/img/user-id.png)
 
              Copy the User ID and paste it into the **User ID** field in the **Add Service Account** modal. The lookahead search should show you the user name You can also optionally set an expiration date:
 
-             ![Adding an impersonated service account](../img/create-impersonated-service-account.png)
+             ![Adding an impersonated service account](/enterprise/img/create-impersonated-service-account.png)
              ::::
              :::::
 
           1. After you click **Submit**, the modal presents the Java Web Token (**JWT**) for the service account. Temporarily save it somewhere secure, as you will not be able to view it again:
 
-             ![Service Account Added](../img/service-account-jwt.png)
+             ![Service Account Added](/enterprise/img/service-account-jwt.png)
```

Comment:
This doesn't work for me?
https://pomerium.com/enterprise/img/service-account-jwt.png

Reply:
I think this is because netlify is doing some url-rewriting. I just want to make sure these are valid / checked?